tests/Applicator/SqlApplicatorTest.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Applicator;
+
+use Lmc\ApiFilter\AbstractTestCase;
+use Lmc\ApiFilter\Entity\Filterable;
+use Lmc\ApiFilter\Entity\Value;
+use Lmc\ApiFilter\Filter\FilterWithOperator;
+
+class SqlApplicatorTest extends AbstractTestCase
```

Comment:
I guess the name inconsistence is fixed in some upcoming commit, right :)?

Reply:
Yes, right in the next PR :) .. renamed